### ISSUE TITLE
[Java] Support RoadMap ?

### ISSUE THREAD

**Issue by HichemMaiza** [microsoft/onnxruntime-genai#328]
Hello Guys, is there any plans to support Java along with the already supported bindings ? 

- Python
- C#
- C/C++

**Comment by natke** [comment]
Yes, there is a plan. We are working on the schedule for this. Converting over to a discussion
